src/assets/lang/zh-tw/lifetime.json

```diff
@@ -9,13 +9,13 @@
       "taiwan": {
         "title": "今天慶祝終身隱私",
         "description": "借國慶折扣永久保護您的文件。",
-        "cta1": "享受 83% 的折扣"
+        "cta1": "享受 ${{percent}}% 的折扣"
```

Comment:
Remove the '%' symbol. The percent is already added in the percent[pathname] variable.